lib/attrsets.nix

```diff
@@ -73,9 +73,9 @@ rec {
        getAttrFromPath ["z" "z"] x
        => error: cannot find attribute `z.z'
   */
-  getAttrFromPath = attrPath: set:
```

Comment:
Btw. I think newer nix versions now also just do one lambda if a function have two arguments: https://github.com/NixOS/nix/pull/5501
Does this affect this optimization in any way?

Reply:
that's purely an optimization of how calling of lambdas is done, you still have one ExprLambda per `:` and one env allocated per ExprLambda. it makes primop calls more efficient, but lambda calls still benefit slightly from this.